src/applications/edu-benefits/10203/manifest.json

```diff
@@ -2,29 +2,5 @@
   "appName": "22-10203 Education benefits form",
   "entryFile": "./edu-benefits-entry.jsx",
   "entryName": "10203-edu-benefits",
-  "rootUrl": "/education/apply-for-education-benefits/application/10203",
-  "template": {
-    "vagovprod": false,
```

Comment:
this form should be disabled in production

Reply:
It will continue to be. https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/registry.json#L553

Reply:
As I mentioned in Slack, [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/12367) changes the way application landing pages are created. We're using the `registry.json` for the content build to create the landing pages.